src/services/AuthService.js

```diff
@@ -0,0 +1,9 @@
+import HttpService from "./HttpService";
+class AuthService extends HttpService {
+  register = async (user) => {
+    const { data } = await this.client.post("/register/", user);
```

Comment:
Rutu izdvajas na pocetku u konstante, da mozes opet da ih koristis, npr.
const ENDPOINTS = {
  USER_REGISTRATION: "/register/",
};
...
 const { data } = await this.client.post(ENDPOINTS.USER_REGISTRATION, user);